cpp/environment/MatchEnv.cc

```diff
@@ -614,9 +611,11 @@ unsigned int EnvironmentCluster::populateEnv(EnvDisjointSet dj)
 
             // label this particle in m_env_index
             m_env_index[particle_ind] = label_ind;
-            for (unsigned int m = 0; m < part_vecs.size(); m++)
+
+            m_point_environments.emplace_back();
+            for (auto& part_vec : part_vecs)
```

Comment:
```suggestion
            for (auto const& part_vec : part_vecs)
```

Reply:
AFAIK, it is best practice to use `const auto` rather than `auto const`, but I suspect in this context it will not make a difference.

Reply:
East const vs. west const is a matter of style. I don't know which is more common in freud's codebase. I tend to fall in the east const (`auto const`) camp.

https://mariusbancila.ro/blog/2018/11/23/join-the-east-const-revolution/
https://hackingcpp.com/cpp/design/east_vs_west_const.html

Reply:
I'm definitely more sympathetic to east const after reading those articles, but after checking the codebase the convention is overwhelmingly west const, so I'll stick with that.